### PR TITLE
split codepoints out from metadata, return unstringified objects

### PR DESCRIPTION
**Pull request by mikemorris**
Refs https://github.com/mapbox/api-core/pull/4922

/cc @jakepruitt

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -21,15 +21,10 @@ fontmachine.makeGlyphs = function(opts, callback) {
     var stack = [];
     fontnik.load(opts.font, function(err, faces) {
         if (err) return callback(err);
-        // temp workaround: fill this out
+
```
Comment (lbud) [inline index.js:24]: @mikemorris @jakepruitt in your font exploration did you manage to find any font files that contain >1 face?

Comment (jakepruitt) [inline index.js:24]: I never found any fonts that threw this error, but it seems like font files [can support more than one face](http://www.freetype.org/freetype2/docs/glyphs/glyphs-1.html) so it's a good check to have. Do any of the fontnik fixtures have multiple faces?

Comment (lbud) [inline index.js:24]: @jakepruitt no, I wasn't able to find any either — hence why I punted by just adding the check but never fixing with any proper iterating ¯_(ツ)_/¯

Comment (mikemorris) [inline index.js:24]: [`.dfont`](https://en.wikipedia.org/wiki/Datafork_TrueType) files can contain multiple fonts, although that's a format that's being phased out. [TrueType Collection](https://en.wikipedia.org/wiki/TrueType#TrueType_Collection) sounds like another older format that may be more common with CJK fonts. I think just keeping the check and error in place is fine.

**Comment by mikemorris** [comment]
Fixed metadata output and updated tests.

File: index.js (hunk 1 above)
Comment (springmeyer) [inline index.js:24]: correct @mikemorris - for ref I've been using `NotoSans-Regular.tcc` for ensuring tcc support in Mapnik does not regress: https://github.com/mapnik/mapnik/commit/5a481fcf2dd563ce38d903d04541de8fd9acfc58

Comment (mikemorris) [inline index.js:24]: Thanks @springmeyer! Opened https://github.com/mapbox/fontmachine/issues/2 to track this further.

**Pull request merged**